webapp/src/routes/Home/ShowLifebanks.js

```diff
@@ -0,0 +1,316 @@
+    height: "40px",
+    width: "40px",
+  },
+  cardTitleContainer: {
+    width: "60%"
+  },
+  cardTitle: {
+    marginLeft: 7,
+    marginTop: 10,
+    width: "100%",
+    fontFamily: "Roboto",
+    fontsize: "16px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "1.5",
+    letterSpacing: "0.15px",
+    textAlign: "left",
+    color: "rgba(0, 0, 0, 0.87)",
+  },
+  cardIconOffer: {
+    position: "absolute",
+    color: "rgba(0, 0, 0, 0.6)",
+    width: 20,
+    height: 20,
+    top: 20,
+    right: 15
+  },
+  cardContent: {
+    marginTop: 10,
+    padding: 0,
+  },
+  cardContentText: {
+    fontFamily: "Roboto",
+    fontsize: "14px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "1.43",
+    letterSpacing: "0.25px",
+    textAlign: "left",
+    color: "rgba(0, 0, 0, 0.6)",
+  },
+  cardActions: {
+    padding: 0,
+    width: "100%",
+    display: "flex",
+    flexDirection: 'column',
+    alignItems: 'flex-end',
+  },
+  cardActionButton: {
+    position: "absolute",
+    bottom: 10,
+    right: 15,
+    fontFamily: "Roboto",
+    fontsize: "14px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "1.14",
+    letterSpacing: "1",
+    textAlign: "center",
+    color: "#121212",
+  }
+}))
+
+const ShowLifebanks = ({ banks, loading, isDesktop }) => {
+  const classes = useStyles()
+
+  const LoadBanks = () => {
+    return (
+      <>
+        {loading &&
+          <Box className={classes.wrapper}>
+            <CircularProgress />
+          </Box>
+        }
+        {!loading && banks.length <= 0 && (
+          <ListItem className={classes.listItem} >
+            <ListItemText
+              primary={
+                <Typography className={classes.listItemPrimaryText} noWrap variant="body2">No blood bank available</Typography>
+              }
+            />
+          </ListItem>
+        )}
+        {!loading && banks.length > 0 && banks.map(bank => (
+          <BankItem
+            key={bank.id}
+            id={bank.id}
+            name={bank.name}
+            description={bank.info.description}
+          />
+        ))}
+      </>
+    )
+  }
+
+  const BankItem = (props) => {
+    return (
+      <ListItem className={classes.listItem} button>
+        <ListItemAvatar>
+          <Avatar>
+            <LocalHospitalIcon />
+          </Avatar>
+        </ListItemAvatar>
+        <ListItemText
+          primary={
+            <Typography className={classes.listItemPrimaryText} noWrap variant="body2">{props.name}</Typography>
+          }
+          secondary={
+            <Typography className={classes.listItemSecondaryText} noWrap variant="body2">{props.description}</Typography>
+          }
+        />
+      </ListItem>
+
+    )
+  }
+
+  BankItem.propTypes = {
+    name: PropTypes.string,
+    description: PropTypes.string,
+  }
+
+  const LoadBanksDesktop = () => {
```

Comment:
When the components only has a return in its JSX content, you can do something like

```
const OfferItem = (props) => (
    JSX HERE
)
```